README.md

```diff
@@ -91,6 +91,140 @@ const filterAlfas = filterCarsByMake("Alfa Romeo");
 cars.filter(filterAlfas); // => [ { make: "Alfa Romeo", model: "Giulia" }, { make: "Alfa Romeo", model: "Stelvio" } ]
 ```
 
+### Either
+**Either** is an interface that wraps values or Errors and can be used to:
+ - handle error cases without throwing and catching errors
+ - provide an alternative to Union types with runtime type safety and no need for typeguards
+```typescript
+import { Either } from "https://deno.land/x/denofun/lib/either.ts";
+
+function handleUnion(stringOrNumber: Either<string, number>): number {
+    const value = stringOrNumber
+        .map((s: string) => s.toUpperCase()) // .map() applies the function to the left value, Either is a Functor
+        .left.map((s: string) => s.toLowerCase()) // but you can also use the .left shorthand for more clarity
+        .right.map((n: number) => n + 1) // .right will provide a way to apply functions to the right value
+        .flatMap((s) => { // Either is also a Monad, .flatMap() is also available on .left and .right
+            const n = parseFloat(s);
+            if (n) {
+                return right(n);
+            } else {
+                return left(s);
+            }
+        });
+    value.get() // => string | number
+    value.coerce(parseFloat) // => number (possibly NaN), also available on .left and .right
+
+    return value.mapEither(parseFloat, (n) => 1 / n) // => number (possibly NaN)
+}
+
+function handleError(stringOrError: Either<string, Error>): string {
+    const safeString: string = stringOrError.catch((err) => err.message);
+    const value: string = stringOrError.try(); // will either throw the error or return the left value
+    const errorValue: stringOrError.right.try(); // will either throw a string or return an error as a value
+
+    return safeString;
+}
+```
+
+### either.error
+**either.error** wraps a value in the Either interface as a right value.
+`JSON.stringify()` will throw the value (ideally an Error)
+```typescript
+import either from "https://deno.land/x/denofun/lib/either.ts";
+// import error from "https://deno.land/x/denofun/lib/either/error.ts"; // is also available
+
+const numberOrError = either.jsonError<number, Error>(new Error("message"));
```

Comment:
Both examples for `error` and `jsonError` use `jsonError` function.

Reply:
fixed :)